src/classes/modules/java.base/java/text/DateFormat.java

```diff
@@ -0,0 +1,83 @@
+package java.text;
```

Comment:
License header: GPL 2 from OpenJDK if we use more than about ten lines of code, Apache v2 license otherwise

Reply:
About 11 lines of code used, hence using the GPL 2 license header.